src/app.js

```diff
@@ -28,7 +28,7 @@ export default class App extends Component {
       return <Start onSubmit={this.createTable} />
     }
     if (this.state.view === 'table') {
-      return this.state.table && <Table seats={this.state.table.seats} event={this.state.table.event} date={this.state.table.date}
+      return this.state.table && <Table seats={this.state.table.seats} event={this.state.table.event} date={this.state.table.date} subtotal={this.state.table.subTotal} tax={this.state.table.tax} quantity={this.state.table.quantity}
```

Comment:
Instead of passing all of the individual properties of `state.table`, simply pass the whole `table` object.

```jsx
<Table table={this.state.table}/>
```